packaging/server_config.json.template.in

```diff
@@ -1,7 +1,11 @@
 {
     "schema_name": "server_config",
     "schema_version": "v@IRODS_CONFIGURATION_SCHEMA_VERSION@",
-    "access_time_ignore_window_in_seconds": 86400,
+    "access_time": {
+        "queue_name": "irods_access_time_manager",
```

Comment:
i see you made an access_time stanza... but now the variables and KW_CFG_QUEUE_NAME have lost their association with access time...

i'm sure you've thought through this, but just noticing here too.

Reply:
Yes, that's intentional, but may change to include ACCESS_TIME as things move forward.

Reply:
Added ACCESS_TIME to names.